libs/net/Dockerfile

```diff
@@ -12,6 +13,10 @@ WORKDIR /src
 COPY . .
 RUN fix_permissions() { while [ $# -gt 0 ] ; do chgrp -R 0 "$1" && chmod -R g=u "$1"; shift; done } && \
     fix_permissions "/src" "/tmp"
+RUN dotnet restore
```

Comment:
This fixes the permission error that Microsoft introduced in their image.  It would be nice to be more specific with our tagged version, but I can't find a documented list of versions.